src/imagery/i.sentinel/i.sentinel.download/i.sentinel.download.html

```diff
@@ -153,59 +111,46 @@ <h2>NOTES</h2>
+Scenes with unavailbe queryable information are filtered out
+(e.g. if a product have the timeliness queryable set as <b>Null</b>,
+then it will be filtered out if you use query="timeliness=NT").
+
 
 <p>
 <em>i.sentinel.download</em> also allows downloading of Sentinel products
-by specifying a (list of) UUID. This operation is performed by the <b>uuid</b>
+by specifying a (list of) ID, where ID refers to the scene name on
+Copernicus Data Space Ecosystem. This operation is performed by the <b>id</b>
 option. Note that this option is mutually exclusive with all other
-filtering options. The format of UUID differs between the Copernicus Open Access
-Hub and the USGS Earth Explorer (see examples below).
+filtering options.
 
 <p>
 In case a Sentinel data download was interrupted, <em>i.sentinel.download</em>
-will continue to download once the command is restarted and the partially
-downloaded file (with <tt>.incomplete</tt> file extention) still present.
-
-<p>
-The <b>output</b> directory is created if not yet available.
+will restart the download for the paritally downloaded data, from scratch,
+once the command is re-excuted.
 
 <p>
-The query string sent to ESA can be shown with <b>--verbose</b>.
-
-<h3>Long Term Archive (LTA)</h3>
-Access to the product URL for data that are no longer available online in the Copernicus Open Access Hub will automatically trigger
-the retrieval of the data from the LTA, the actual download may be initiated once the data are restored
-(within minutes to hours). Full details are available in the updated
-<a href="https://scihub.copernicus.eu/userguide/" target="_blank">user guide</a>,
-section <a href="https://scihub.copernicus.eu/userguide/LongTermArchive" target="_blank">Long Term Archive (LTA) Access</a>.
-
-In this <a href="https://github.com/sentinelsat/sentinelsat/issues/358#issuecomment-615895388" target="_blank">issue of sentinelsat</a>,
-it is stated (as of 2020):<br>
-<em>Please be advised that a user quota on the maximum number of requests per hour per user is set. Note that if you</em>
-<ul>
-  <li><em>retrieve data directly from the GUI, you can retrieve only 1 product per half hour while</em></li>
-  <li><em>from APIHub 20 products each 12h</em></li>
-</ul>
+The <b>output</b> directory is created if not yet available. Default is current working directory.
 
+<!-- Not sure if Data Space Ecosystem doesn't have an archive, but apparently not -->
+<h3>Long Term Archive (DEPRECATED)</h3>
 To work with LTA you need to define the <b>sleep</b> parameter (and you can set the maximum number of retries with
 the <b>retry</b> parameter) to retry a download after the <b>sleep</b> value in minutes.
 
-<p>
-As an alternative to triggering re-upload Sentinel-2 data from the LTA, scenes can be downloaded
-from <b>USGS Earth Explorer</b> or <b>Google Cloud Storage</b> where all data remain online and directly accessible.
-However, this applies only to Sentinel-2 data in levels 1C (USGS and GCS) and 2A (only GCS).
```

Comment:
It would be great if you could structure examples by Satellte (h3 for Sentinel-1, -2, and -3...).